app/src/main/java/org/oppia/android/app/topic/TopicActivity.kt

```diff
@@ -117,7 +116,6 @@ class TopicActivity :
     startActivity(
       ExplorationActivity.createExplorationActivityIntent(
         this,
-        profileId,
```

Comment:
Here and routeToResumeLesson below, at which point is the intent decorated with the profileId as a replacement?